### DISCUSSION TITLE
status of this project

### DISCUSSION THREAD

**Discussion by guykatz** [atlassian-labs/nadel#200] [Q&A]
Hi guys;
how would you define the status of this project? can it b used in production? I remember asking about a year ago and I was told its not in the focus of the gql-java team. is it still the case?
thanks!

**Comment by joostfarla** [comment]
Hi, I was just wondering the same thing! At a first glance (commit activity), this project seems active. But looking at the release history, there hasn't been any stable release (yet). I would be very interested in the status of this project and where contributions might be needed to bring this project to the next level. Thanks!
